acceptance/tests/i18n/disable_option_disable_i18n.rb

```diff
@@ -0,0 +1,44 @@
+test_name 'C100562: verify that disable_i18n can be set to false and have translation still enabled' do
+  confine :except, :platform => /^eos-/ # translation not supported
+  confine :except, :platform => /^cisco_/ # translation not supported
+  confine :except, :platform => /^cumulus/ # translation not supported
+  confine :except, :platform => /^solaris/ # translation not supported
+  confine :except, :platform => /^aix/ # QENG-5283 needed for this to work
+
+  tag 'risk:medium',
+      'audit:acceptance'
+
+  require 'puppet/acceptance/i18n_utils'
+  extend Puppet::Acceptance::I18nUtils
+
+  language = 'ja_JP'
+  agents.each do |agent|
+
+    puppet_conf = agent.tmpfile('puppet_conf_test')
+    config      = <<-EOM
+    [user]
+    disable_i18n = false
```

Comment:
I don't feel that we need another test for this, it's the default behavior and covered by check_puppet_run_message.

Reply:
Hmmm. Yeah, it seems reasonable to remove the test as cover it in the other and if the default changes the tests will have to change to support the behavior change.